server/controllers/article_controller.js

```diff
@@ -11,8 +11,8 @@ export const createArticle = (uri, groups, score) => {
   const article = new Article();
   article.uri = uri;
   article.groups = groups;
-  article.avgUserScore = (score || 1);
-  article.numShares = (score ? 1 : 0);
+  article.avgUserScore = (score ? score : 1);
```

Comment:
Hm interesting, idk why that would happen but we can leave it if it fixes the issue!